selftest/create-map/main.go

```diff
@@ -36,10 +36,6 @@ func main() {
 	}
 	defer bpfModule.Close()
 
-	bpfModule.BPFLoadObject()
-	opts := bpf.BPFMapCreateOpts{}
-	opts.Size = uint64(unsafe.Sizeof(opts))
```

Comment:
It seems a leftover change not related to this PR.

Reply:
Yeah, tt's dead code I saw while working in the PR